csympy/lib/csympy.pxd

```diff
@@ -12,6 +12,17 @@ cdef extern from 'gmpxx.h':
     cdef cppclass mpq_class:
         mpq_class()
 
+cdef extern from "<set>" namespace "std":
+    cdef cppclass set[T, U]:
```

Comment:
Cython's `set.pxd` file does not support two template arguments to `set`, so it is defined here with minimal methods.